components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  enabled: boolean;
+}
+
+export default function Button({ children, enabled }: Props) {
```

Comment:
Using an arrow function + `FC` type will make this type definition a bit simpler
```suggestion
const Button: FC<PropsWithChildren<Props>> = ({ children, enabled }) => {
```